USER-FEEDBACK-SESSIONS-HOW-TO.md

```diff
@@ -0,0 +1,65 @@
+# Node.js User Feedback Sessions: A Guide
+
+## Announcing Node.js User Feedback Sessions
+
+The ultimate goals of the Node.js User Feedback Sessions are to:
+
+- Facilitate connections between the wider network of Node.js contributors and users
+- Encourage Node.js users to contribute back to the ecosystem by showing them how
+- Create a feedback loop for Node.js users to know where to send feedback on release and upgrade issues and package problems.
+
+We want to know how our community is doing and how we can provide users with everything they need to succeed.
+
+
+## What is a Node.js User Feedback Session and Who Runs Them?
+
+If you are an organizer or speaker of a Node.js meetup, this is a guide on how to run a Node.js user feedback session.
+
+User feedback sessions can either be run as part of a meetup or as a standalone meetup.
+
+The user feedback sessions take approximately 60 minutes and are complemented by slides focused on topics such as Node.js recent updates or releases. These slides will be provided to you as well.
+
+At these sessions, we are hoping that organizers and speakers will provide help and facilitate learning on how users can submit feedback to problems they are having with Node.js. 
+
+## How to Run a Node.js User Feedback Session - Getting Started
+
+** Agree to Run a User Feedback Session **
+
+Does helping people interest you? That’s the first question you should be asking yourself. If the answer is yes, running a session is the job for you! 
+
+- If you run a local meetup, post your meetup on meetup.com to get started.
+- If you don’t run your local meetup, then find a local meetup that will host you as a speaker for this user feedback session. If you would like, you can augment the user feedback session with a talk on Node.js as well! This is highly encouraged.
+- Tell the Node.js User Feedback Group that you are running a session
+- Use the [issue template](https://github.com/nodejs/user-feedback/tree/master/.github/ISSUE_TEMPLATE) labeled User Feedback Session to create an issue
+- Post it as an issue under [issues](https://github.com/nodejs/user-feedback/issues)
+- Review session materials and where to find help
+-- We have created session materials for you that you can find [here](https://docs.google.com/presentation/d/10ps715QthCsdTE_YmIRvfbYZC8onhQ7fPvlaYVISBAA/edit#slide=id.g50adb52472_0_25). 
+-- Have questions? Create an issue in the repo [here](https://github.com/nodejs/mentorship/issues) so we can help!
+
+## During the Node.js User Feedback Session - Running the Session
+
+- Use the introduction slides found [here](https://docs.google.com/presentation/d/10ps715QthCsdTE_YmIRvfbYZC8onhQ7fPvlaYVISBAA/edit#slide=id.g50adb52472_0_25) (WIP).
+- Introduce the structure of the project and where people can get feedback
+- Introducing the project will be by way of opening an issue on GitHub underneath and having people submit PR’s.
```

Comment:
Not sure this is quite right.
Maybe

- Feedback will be captured through issues in GitHub.  There should be open issues related to the questions you are going to ask in the session.  In some cases these  may be existing issues, in others you may need to co-ordinate with the project to get them opened in the right place.